### ISSUE TITLE
PDF export changes format of date in resulting PDF file

### ISSUE THREAD

**Issue by zocker-160** [ONLYOFFICE/DocumentServer#1356]
**Do you want to request a *feature* or report a *bug*?**
bug

**What is the current behavior?**
![grafik](https://user-images.githubusercontent.com/36563538/126909894-94346199-44b9-4978-8de4-89498cfe2a1b.png)

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
Create a new document with a cell formated as date, type a valid date in and export it as PDF. It seems like the German date format gets converted into a different one.

**What is the expected behavior?**
the .xslx and the PDF should have use the exact same formats

**Which versions of DesktopEditors, and which OS are affected by this issue? Did this work in previous versions of DesktopEditors?**
- 6.3.1.56 on Ubuntu Linux (DEB)

**additional note**
the same file converted to PDF using LibreOffice does not have this issue

**Comment by ShockwaveNN** [comment]
Moving this issue to DocumentServer repo, since it's our base product and problem actual there too

**Comment by 2bmade** [comment]
This bug is still present
<img width="975" alt="Screenshot 2022-08-14 at 11 28 36" src="https://user-images.githubusercontent.com/94042657/184530988-95e062ab-7817-44f1-a6eb-eb55d59c50dd.png">

**Comment by ShockwaveNN** [comment]
@2bmade Yes, this issue is still unresolved and currently we got not ETA of fix, sorry

**Comment by dvaceov** [comment]
Hello,
I'm re-launching this issue to inform you of a date conversion bug between Excel (latest version: (2312 Build 16.0.17126.20132)) and onlyoffice documentserver (version 7.4.1.36). This problem also appears on the Excel 2016 version.

I'm using the French version of Excel and, in the language settings, I've specified that the office display language settings and the office language creation and verification settings should be set to French.

![image](https://github.com/ONLYOFFICE/DocumentServer/assets/152510201/98e34e0f-8ec9-4686-8a7e-2dd228c601f9)



Similarly, on the onlyoffice documentserver settings, the Workspace, Regional and Verification settings are set to work in the French language and region.

![image](https://github.com/ONLYOFFICE/DocumentServer/assets/152510201/b21e20f6-fd8f-4a8e-b38e-887e76f9d9b8)


The date format is set in the same way for Excel and onlyoffice (dd/mm/yyyy):
 
Excel:

![image](https://github.com/ONLYOFFICE/DocumentServer/assets/152510201/8cec54b1-e070-44b4-9b21-820ad684e01e)


OnlyOffice:

![image](https://github.com/ONLYOFFICE/DocumentServer/assets/152510201/3b9976c6-d159-42c9-a43b-312b186f1ae1)


Then, when I edit a document in Excel, I specify a date in dd/mm/yyyy format:

![image](https://github.com/ONLYOFFICE/DocumentServer/assets/152510201/3e492b53-9d0c-4bae-94fb-09d7be96f98c)


If I then open the excel document in documentserver, the modification appears in the correct format:

![image](https://github.com/ONLYOFFICE/DocumentServer/assets/152510201/da2c24bc-c0da-4066-a9a3-4f5b30edccab)


However, when I convert the .xlsx document to .pdf format with onlyoffice, the date format changes:

![image](https://github.com/ONLYOFFICE/DocumentServer/assets/152510201/95a747c3-7c70-4f29-8627-41b5746789b8)


This problem does not appear when the xlsx document is edited only with onlyoffice, nor does it appear if the document is edited with libre office and then imported into documentserver.

Thank you in advance for any solutions you can find to this problem.
